src/Analyzers/VisualBasic/Tests/MakeDeclarationPartial/MakeDeclarationPartialTests.vb

```diff
@@ -0,0 +1,156 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions
+Imports Microsoft.CodeAnalysis.Testing
+Imports Microsoft.CodeAnalysis.VisualBasic.MakeDeclarationPartial
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.MakeDeclarationPartial
+    <Trait(Traits.Feature, Traits.Features.CodeActionsMakeDeclarationPartial)>
+    Public Class MakeDeclarationPartialTests
+        Private Class TestVerifier
+            Inherits VisualBasicCodeFixVerifier(Of EmptyDiagnosticAnalyzer, VisualBasicMakeDeclarationPartialCodeFixProvider).Test
+        End Class
```

Comment:
Did you see this pattern somewhere? I don't think I saw it in the repo before.

Reply:
The example of multiple-document VB test you provided before uses it: https://github.com/dotnet/roslyn/blob/208c7da2a7e3175e1a7cab42ff6742ed19d5d130/src/EditorFeatures/VisualBasicTest/InlineMethod/VisualBasicInlineMethodTests.vb#L13-L14

Reply:
Ah I see. But this one has some additional functionality in the class. For the case in this PR, the class doesn't have any additional functionality. So using `VerifyVB` as most other VB tests should work fine.